passwd.master

```diff
@@ -34,3 +34,4 @@ geoclue:*:114:122::/var/lib/geoclue:/usr/sbin/nologin
 speech-dispatcher:*:115:29:Speech Dispatcher,,,:/var/run/speech-dispatcher:/usr/sbin/nologin
 app-manager:*:116:123:Endless App Manager:/var/endless:/usr/sbin/nologin
 hplip:*:117:7:HPLIP system user,,,:/var/run/hplip:/usr/sbin/nologin
+kalite:*:118:8:KA Lite Helper system user,,,:/var/lib/kalite/:/bin/sh
```

Comment:
Group should match the newly created 'kalite' group (124).
Shell can probably be nologin, too.